index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="//use.fontawesome.com/releases/v5.0.7/css/all.css"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <nav class="container">
+      <header class="toolbar">
+        <div class="logo">
+          <div class="placeholder">
+            <h3>Eshetu Melaku</h3>
+          </div>
+        </div>
+
+        <div class="normal-button">
+          <div id="icon-menu">
+            <div class="menu-line"></div>
+            <div class="menu-line"></div>
+            <div class="menu-line"></div>
+          </div>
+        </div>
+      </header>
+    </nav>
+
+    <section class="presentation">
+      <h1 class="title-presentation">Hi There. I'm Melaku</h1>
+      <span>I’am a Software Developer</span>
+      <p>
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </p>
```

Comment:
Kindly try to match the `line-height` for the paragraph with the Figma design.

Reply:
Thanks again, I just did that also.